File: Java Script/Day 10/this.js
const student ={
    name: "keshav",
    age:18,
    eng:95,
    math:59,
    phy:97,
   

}

function getAvg(){
    console.log(this);
    // let avg=(this.eng + this.math + this.phy)/3;
    // console.log(avg);
}
getAvg();

File: this.cpp
#include<iostream>
using namespace std;

class Student {
    int age;
    string name;
    public:
    Student(int age, string name){
        this->age=age;
        this->name=name;

    }
    void display(){
        cout<<"Name: "<<name<<endl;
        cout<<"Age: "<<age<<endl;

    }

};

int main () {

        Student s1(18,"keshav");
        s1.display();


    return 0;
}